packages/plugin-audio/web/src/modals/AudioSettings.tsx

```diff
@@ -33,10 +34,8 @@ const AudioSettings = ({
   blockKey,
   getComponentData,
 }) => {
-  const useUploadService = !!experiments?.useUploadContext?.enabled;
-  const { uploadService }: UploadContextType = useUploadService
-    ? useContext(UploadServiceContext)
-    : {};
+  const useUploadService = !!experiments?.tiptapEditor?.enabled;
+  const { uploadService }: UploadContextType = useUploadService ? useContext(UploadContext) : {};
```

Comment:
```suggestion
  const { uploadService } = useContext(UploadContext);
```
The condition is useless since this component is only rendered on Tiptap